src/main/java/Formatter.java

```diff
@@ -0,0 +1,24 @@
+import java.math.BigDecimal;
+
+public class Formatter {
+    public static String getCurrencyText(BigDecimal price) {
+        String roundPrice = price.toBigInteger().toString();
+        String currency;
+        if (roundPrice.length()>1){
+            int lastDigits = Integer.parseInt(roundPrice.substring(roundPrice.length() - 2));
+            if(lastDigits>10&&lastDigits<20){
```

Comment:
🍏 Для повышения читабельности:
```suggestion
            if (lastDigits > 10 && lastDigits < 20) {
```